### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by mardiros**
Create a documentation for the wagtail-localize-panel

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.. wagtail-localize-panel documentation master file, created by
+   sphinx-quickstart on Thu Dec  9 08:54:19 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Wagtail OAuth2
```
Comment (Themimitoof) [inline docs/source/index.rst]:
```suggestion
Wagtail Localize Panel
```

File: docs/source/users/installation.rst [hunk 2]
```diff
@@ -0,0 +1,99 @@
+Install and configure wagtail-localize-panel
+============================================
+
+
+Installation
+------------
+
+`wagtail-localize-panel` is available on pypi, you can install with your favorite
```
Comment (Themimitoof) [inline docs/source/users/installation.rst]:
```suggestion
``wagtail-localize-panel`` is available on pypi, you can install with your favorite
```

Instead no (and idem for the rest of the doc)?

**Review by Themimitoof — CHANGES_REQUESTED** [review]

File: docs/source/users/configuration.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+   :alt: List users.
+
+And add the new role tab of the user, check the box of the new group and
+click the save button.
+
+.. image:: ./screen_7.png
+   :alt: Add the role to the user user.
+
+
+Now users in the group have the panel of translations in there home page:
```
Comment (yanndinendal) [inline docs/source/users/configuration.rst]:
```suggestion
Now users in the group have the panel of translations in their home page:
```

File: docs/source/users/configuration.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+.. image:: ./screen_8.png
+   :alt: Add the role to the user user.
```
Comment (yanndinendal) [inline docs/source/users/configuration.rst]:
```suggestion
   :alt: Add the role to the user.
```

File: docs/source/users/configuration.rst [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+Edit users under the menu `Settings / Users`
+
+.. image:: ./screen_6.png
+   :alt: List users.
+
+And add the new role tab of the user, check the box of the new group and
+click the save button.
+
+.. image:: ./screen_7.png
+   :alt: Add the role to the user user.
```
Comment (yanndinendal) [inline docs/source/users/configuration.rst]:
```suggestion
   :alt: Add the role to the user.
```

File: docs/source/users/configuration.rst [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+
+Now users in the group have the panel of translations in there home page:
+
+.. image:: ./screen_8.png
+   :alt: Add the role to the user user.
+
+if the checkbox `Receive email on new translations` is checked,
+every time the page in the source lang is published, an email will be
+sent to users to prevent for the incoming translations.
```
Comment (yanndinendal) [inline docs/source/users/configuration.rst]:
```suggestion
sent to users to notify them of the incoming translations.
```

File: docs/source/users/configuration.rst [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+
+.. important::
+
+   This plugin automatically "synchronize translations", so it is now useless
```
Comment (yanndinendal) [inline docs/source/users/configuration.rst]:
```suggestion
   This plugin automatically does the "synchronize translations", so it is now useless
```

File: docs/source/users/installation.rst (hunk 2 above)
Comment (yanndinendal) [reply docs/source/users/installation.rst]:
```suggestion
``wagtail-localize-panel`` is available on pypi, you can install it with your favorite
```

File: docs/source/users/installation.rst [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+
+Django's Configuration
+----------------------
+
+INSTALLED_APPS
+~~~~~~~~~~~~~~
+
+`wagtail-localize-panel` is a Django application that must be installed,
+so it must be in the `INSTALLED_APPS` list of your settings module.
+And, to work, it requires other app that are probably already installed.
```
Comment (yanndinendal) [inline docs/source/users/installation.rst]:
```suggestion
And, to work it requires other apps that are probably already installed.
```

File: docs/source/users/installation.rst [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+
+Here is the lists of the minimum installed app.
```
Comment (yanndinendal) [inline docs/source/users/installation.rst]:
```suggestion
Here is the lists of the minimum apps to install.
```

File: docs/source/users/installation.rst [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+      "wagtail.locales",
+      "wagtail_localize_panel",
+      # ...
+   ]
+
+
+TEMPLATES
+~~~~~~~~~
+
+`wagtail-localize-panel` render an admin template `workflow_pages_to_translate.html`
```
Comment (yanndinendal) [inline docs/source/users/installation.rst]:
```suggestion
``wagtail-localize-panel`` renders an admin template ``workflow_pages_to_translate.html``
```

File: docs/source/users/installation.rst [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+for the translators.
+
+The template is provided using django template system in the package,
+the settings `APP_DIRS` must be set to `True` in order to render it.
```
Comment (yanndinendal) [inline docs/source/users/installation.rst]:
```suggestion
the setting ``APP_DIRS`` must be set to ``True`` in order to render it.
```

File: docs/source/users/installation.rst [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+         "APP_DIRS": True,
+      }
+   ]
+
+
+SETTINGS
+~~~~~~~~
+
+The wagtail site appname that needs to be translated must be passed
+in parameter in a parameter LOCALIZE_PANEL_APP_NAME
```
Comment (yanndinendal) [inline docs/source/users/installation.rst]:
```suggestion
in parameter in a parameter ``LOCALIZE_PANEL_APP_NAME``.
```

File: docs/source/users/installation.rst [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+
+::
+
+   LOCALIZE_PANEL_APP_NAME = "<appname>"
+
+
+Wagtail email must also properly be set, for example:
```
Comment (yanndinendal) [inline docs/source/users/installation.rst]:
```suggestion
Wagtail email must also be properly set, for example:
```

File: docs/source/users/installation.rst [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+   EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+   EMAIL_HOST = "smtp.example.net"
+   EMAIL_PORT = 25
+   WAGTAILADMIN_NOTIFICATION_FROM_EMAIL = "wagtail-admin-email@example.net"
+
+
+
+.. note::
+
+   If the django user model has been redefined, e.g. not `auth_`,
```
Comment (yanndinendal) [inline docs/source/users/installation.rst]:
```suggestion
   If the django user model has been redefined, e.g. not ``auth_``,
```

File: docs/source/users/installation.rst [hunk 15]
```diff
@@ -0,0 +1,99 @@
...
+   then the prefix must be set using the setting `LOCALIZE_PANEL_AUTH_PREFIX`
```
Comment (yanndinendal) [inline docs/source/users/installation.rst]:
```suggestion
   then the prefix must be set using the setting ``LOCALIZE_PANEL_AUTH_PREFIX``
```

File: docs/source/users/installation.rst [hunk 16]
```diff
@@ -0,0 +1,99 @@
...
+      pip install wagtail-localize-panel
+
+Django's Configuration
+----------------------
+
+INSTALLED_APPS
+~~~~~~~~~~~~~~
+
+`wagtail-localize-panel` is a Django application that must be installed,
+so it must be in the `INSTALLED_APPS` list of your settings module.
```
Comment (yanndinendal) [inline docs/source/users/installation.rst]:
```suggestion
``wagtail-localize-panel`` is a Django application that must be installed,
so it must be in the ``INSTALLED_APPS`` list of your settings module.
```

**Review by yanndinendal — APPROVED** [review]
:+1:

**Pull request merged**